Tests/KomondorTests/KomondorTests.swift

```diff
@@ -1,4 +1,4 @@
-@testable import komondor
+@testable import Komondor
```

Comment:
When trying to commit I receive an error:

```
Komondor/Tests/KomondorTests/KomondorTests.swift:1:18: error: no such module 'Komondor'
@testable import Komondor
                 ^
note: module 'Komondor' is the main module of an executable, and cannot be imported by tests and other targets
```

I could extract all code except `main.swift` into a separate target (e.g. `KomondorKit`) if you like.